spk/python310/src/requirements.txt

```diff
@@ -0,0 +1,52 @@
+# Basic wheels
+setuptools==58.3.0
+pip==21.3.1
+wheel==0.37.0
+
+# cryptography dependencies
+idna==3.3
+ipaddress==1.0.23
+
+# Pure-python via spksrc.wheel.mk
+# commonly used wheels
+appdirs==1.4.4
+attrs==21.2.0
+beautifulsoup4==4.10.0
+certifi==2021.10.8
+chardet==4.0.0
+click==8.0.3
+distlib==0.3.3
+filelock==3.3.2
+Flask==2.0.2
+future==0.18.2
+html5lib==1.1
+idna==3.3
+importlib-metadata==4.8.1
```

Comment:
If a en ipaddress listed twice.

Reply:
removing duplicate as well, thnx for noticing.